tests/README.md

```diff
@@ -0,0 +1,57 @@
+Redis Test Suite
+================
+
+The normal execution mode of the test suite involves starting and manipulating
+local `redis-server` instances, inspecting process state, log files, etc.
```

Comment:
do we wanna document `--single`, `--loop`, `--only`, `--clients` etc?
seems a bit odd that we mainly focus on tags (although these are not mentioned in `--help` so that's a good reason)

Reply:
I think it's sufficiently documented in the help.